src/components/MainContentWrapper.tsx

```diff
@@ -72,11 +75,14 @@ const MainContentWrapper = ({ children }: PropsWithChildren) => {
 
         let totalDebt = AmountMath.makeEmpty(debtBrand);
         let totalLocked = AmountMath.makeEmpty(debtBrand);
-        let numVaults = 0;
+        let numVaultsWithDebt = 0;
 
         [...metrics.values()].forEach(
           ({ numActiveVaults, totalCollateral, totalDebt: managerDebt }) => {
-            numVaults += numActiveVaults;
+            // This metric misleadingly only counts vaults with non-zero debt,
+            // even if they're 'active'.
```

Comment:
UNTIL https://github.com/Agoric/agoric-sdk/issues/9463

(closed rn but it will probably happen eventually)

Reply:
Added the issue link